site/content/en/docs/Guides/feature-stages.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Feature Stages"
+linkTitle: "Feature Stages"
+date: 2019-09-26T01:20:41Z
+weight: 5
+description: >
+  This page provides a description of the various stages that Agones features can be in, and the
+  relative maturity and support level expected for each level.
+---
+
+A feature within Agones can be in `Alpha`, `Beta` or `Stable` stage.
+
+## Feature Gates
+
+`Alpha` and `Beta` features can be enabled or disabled through the `agones.featureGate` configuration option 
+that can be found in the [Helm configuration]({{< ref "/docs/Installation/helm.md#configuration" >}}) documentation.
```

Comment:
This implies that if you install via other mechanisms (e.g. install.yaml) that you cannot toggle feature gates. This pushes us a bit further down the "helm is the right way" path, which is probably ok but we should be cognizant that we are doing it.

Reply:
Yeah that is a good point - but you really can't configure much with the install.yaml -- we likely should be clear that install.yaml is good for a "getting started", and helm is good for "production" (another reason to rejig the install pages)

Reply:
sgtm.